tgui/packages/tgui/interfaces/BankingComputer/CrewAccounts.tsx

```diff
@@ -170,32 +173,36 @@ export const CrewAccounts = (props: CrewAccountsProps, context) => {
 
   };
 
-  const searchAndSortCrewAccounts = (filter: string, text: string, sortConfig: CrewColumnSortConfig) => {
+  const searchAndSortCrewAccounts = (
```

Comment:
Looks like this is a util function rather than part of the render, if this doesn't depend on props/state then extract out of the component so it's not regenerated each render (plus makes the code more bitesize).